### PR TITLE
Update for KCSNA23

### PR DESCRIPTION
**Pull request by tokt**
Website update for the KCSNA 2023 in Chicago

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] Update for KCSNA23

There is still at least one TODO in the PR. @tokt I assume you plan to fix that? Feel free to mark this ready to review once you have.

File: content/en/events/2023/kcsna/_index.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/_index.md:26]: All the instances of community@kubernetes.io we'll want to replace with summit-team@kuberrnetes.io (or whatever it is if we want to rename it)

File: content/en/events/2023/kcsna/_index.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/_index.md:42]: All the TODOs can probably just be removed until we have more info, or replaced with 'TBD'

File: content/en/events/2023/kcsna/faq.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/faq.md:66]: Not necessarily something to add to the list, but an FYI for reg we should also accept etcd as they're progressing towards being a part of k8s.

**Review by mrbobbytables — COMMENTED** [review]
One other thing that should def go in there with this going live is update the redirects so k8s.dev/summit takes you to this event:

https://github.com/kubernetes/contributor-site/blob/master/static/_redirects

File: content/en/events/2023/kcsna/_index.md
Comment (tokt) [reply content/en/events/2023/kcsna/_index.md:42]: Thank you! I'll remove old links/info and put TBD until we have more information, would that be OK?

File: content/en/events/2023/kcsna/faq.md
Comment (tokt) [reply content/en/events/2023/kcsna/faq.md:66]: Totally forgot, thank you!! Will also update the registration playbook to include etcd folks check, will do it later this/next week.

File: content/en/events/2023/kcsna/faq.md
Comment (reylejano) [inline content/en/events/2023/kcsna/faq.md:94]:
```suggestion
full day from 11am to 4pm.
```
I'm not sure what time the awards ceremony starts but in the past it has been around 4pm

File: content/en/events/2023/kcsna/faq.md
Comment (reylejano) [inline content/en/events/2023/kcsna/faq.md:73]:
```suggestion
If you are a SIG lead or subproject lead and there is someone that is not yet
```
not sure if the change from subproject owner to subproject lead is official

File: static/_redirects
Comment (reylejano) [inline static/_redirects:40]: Can ContribEx create a KCSNA 2023 Notes folder and replace the link here

**Comment by pnbrown** [comment]
Checking on the status here. What do we need for the site to go up?

**Comment by stmcginnis** [comment]
/lgtm

No typos or errors that I noticed.

File: content/en/events/2023/kcsna/_index.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/_index.md:13]:
```suggestion
The North America summit will be a full contributor summit event taking place on November 6th
```
This is a full event as we have the CFP going this time. Not necessarily a blocker to this getting merged, but something we'd want to follow up on before blasting comms out.

File: content/en/events/2023/kcsna/_index.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/_index.md:41]:
```suggestion
### Call for Papers

The CFP will be launching soon, check back here for more info!

```

The CFP link points to the one from EU, need to update things with the new one when its available.

File: content/en/events/2023/kcsna/registration.md
Comment (mrbobbytables) [reply content/en/events/2023/kcsna/registration.md:24]: The reg link is for EU. Need to update it once the NA one becomes available.

File: content/en/events/2023/kcsna/schedule.md
Comment (mrbobbytables) [inline content/en/events/2023/kcsna/schedule.md:23]:
```suggestion
If you have a topic in mind, please check back soon - we'll launch a tracking issue for submissions closer to the event.
```
Issue link points to EU

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks @tokt , looks pretty good - just a few updates that are needed before this can be merged (references to EU)

File: content/en/events/2023/kcsna/faq.md
Comment (jberkus) [reply content/en/events/2023/kcsna/faq.md:66]: Suggested text:

This year, Etcd contributors with 50 or more contributions in the last year are also invited to attend, in anticipation of [Etcd becoming a Kubernetes SIG](https://github.com/kubernetes/community/pull/7372).

File: content/en/events/2023/kcsna/registration.md
Comment (jberkus) [inline content/en/events/2023/kcsna/registration.md:8]:
```suggestion
Registration is open to all members of one of the [Kubernetes orgs], Etcd contributors with 50 or more contributions, or sponsored attendees.
```

File: content/en/events/2023/kcsna/faq.md
Comment (mrbobbytables) [reply content/en/events/2023/kcsna/faq.md:66]: I think we can just flag org membership as the general baseline, but if they aren't an org member there yet and have contributed a fair amount to still reach out to us.

File: static/_redirects
Comment (tokt) [reply static/_redirects]: Removed old notes link and updated EU summit page.

File: content/en/events/2023/kcsna/_index.md
Comment (tokt) [reply content/en/events/2023/kcsna/_index.md]: We now have proper links for Reg and CFP

File: content/en/events/2023/kcsna/_index.md
Comment (tokt) [reply content/en/events/2023/kcsna/_index.md]: Done. Thank you!!

File: content/en/events/2023/kcsna/faq.md
Comment (jberkus) [reply content/en/events/2023/kcsna/faq.md:66]: The org membership program for Etcd is brand-new, and most people aren't signed up for it yet.   Including some of the maintainers.  I can provide a list of < 50contributions.

Comment (tokt) [reply content/en/events/2023/kcsna/faq.md:66]: @jberkus Will it make sense to include a devstats link so folks can check eligibility before registering? (i.e. If I made some etcd contributions last summer, am I invited or not?). 
Or I can put something like "Etcd contributors are [now](https://github.com/kubernetes/community/pull/7372) eligible to be part of Kubernetes org and more than welcome at the event"?

Comment (jberkus) [reply content/en/events/2023/kcsna/faq.md:66]: Gah, this requires more explanation/details.  It's going to be easier to just send them to the Etcd Devs directly.  Maybe just:

This year, Etcd project Members and regular contributors are eligible to attend the Contributor Summit.  More information on this will be sent to the Etcd Developer mailing list.

File: content/en/events/2023/kcsna/registration.md
Comment (jberkus) [inline content/en/events/2023/kcsna/registration.md:7]: based on our discussion, change this:

```suggestion
Registration is open to all members of one of the [Kubernetes orgs], qualified Etcd contributors, or sponsored attendees.
```

**Comment by reylejano** [comment]
/lgtm
@tokt If this is ready, please remove the [WIP] prefix in the title

**Comment by jberkus** [comment]
Looks good, I'm merging

/approve

**Pull request merged**
